backend/src/main/java/com/yigongil/backend/domain/study/Study.java

```diff
@@ -165,6 +167,14 @@ public void validateMaster(Member candidate) {
         currentRound.validateMaster(candidate);
     }
 
+    public RoundOfMember findCurrentRoundOfMemberOwn(Member member) {
```

Comment:
p3: 
```suggestion
    public RoundOfMember findCurrentRoundOfMembeBy(Member member) {
```
로 통일해줘도 좋을 것 같아요 !